app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -173,7 +178,15 @@ class ThreadListAdapter @Inject constructor(
         }
     }.getOrDefault(super.getItemId(position))
 
-    private fun CardviewThreadItemBinding.displayThread(thread: Thread) {
+    fun getItemPosition(threadUid: String): Int? {
+        return dataSet
+            .indexOfFirst { it is Thread && it.uid == threadUid }
+            .let { if (it == -1) null else it }
```

Comment:
When reading this line afterhand, I think it would be good to explicit the it with "position"